commit/operator_withCUDA.cuh

```diff
@@ -0,0 +1,194 @@
+#include <cuda.h>
```

Comment:
Move the file inside `operator`

Reply:
I cannot move these files to the 'operator' folder because COMMIT compiles all the files in this folder using gcc. So, an error raises for the .cu files. I can create another folder called 'operator_cuda'.